mavlink/modules/flight_input_device.py

```diff
@@ -0,0 +1,44 @@
+"""
+Creates a FlightInputDevice using dronekit which abstracts MavLink data
+Formats odometry data to dictionary 
+"""
+
+import dronekit
+import modules.drone_odometry as drone_odometry
+
+class FlightInputDevice: 
+    """
+    Wrapper for flight input
+    """
+
+    def __init__(self, address: "str"):
+        """
+        Connection address: tcp address (e.g. "tcp:127.0.0.1:14550")
+        Establishes address to host and stores inside instance self.drone
+        """
+        self.drone = dronekit.connect(address, wait_ready = True)
+        assert self.drone is not None
```

Comment:
If this may possibly fail then use the create() design pattern similar to the classes in drone_odometry.py